### DISCUSSION TITLE
Can't get Sonoff ZBDONGLE-P working in docker

### DISCUSSION THREAD

**Discussion by mygithubusername112** [Koenkk/zigbee2mqtt#19396] [Devices]
Hi all, my previous USB coordinator (Nortek) stopped working and after searching for recommended adapters I got the Sonoff ZBDONGLE-P.  I followed the instructions for flashing onto the latest firmware (I regret this, in hindsight, as I'm not sure it matters).

Anyways, finally got z2m to start up after working through issues, but it doesn't seem to be "seeing" the coordinator properly.  The UI "devices" tab shows nothing - I feel like the coordinator should be in here but can't remember 100%.  Pairing devices does not work.  If I wake a device to pair, I see a log message like "Entity '0x84b4dbfffeff41c1' is unknown".

I'm attaching the logs, and it looks like it finds the coordinator, it reports the firmware version, etc.  But nothing else helpful aside from these "unknown" logs.  Also attaching my compose file and the config yaml.

I'd really appreciate any tips!

[z2m_log.txt](https://github.com/Koenkk/zigbee2mqtt/files/13074056/z2m_log.txt)

[docker-compose.yml.txt](https://github.com/Koenkk/zigbee2mqtt/files/13074096/docker-compose.yml.txt)

[configuration.yml.txt](https://github.com/Koenkk/zigbee2mqtt/files/13074097/configuration.yml.txt)

**Comment by mygithubusername112** [comment]
Another note, I just tried this all on bare linux (same machine, not docker this time) and exact same result.

**Reply by francisp2** [reply]
You did remove the 'adapter: ezsp' line ?

**Reply by mygithubusername112** [reply]
Yes, it's not in my config yaml.

**Reply by francisp2** [reply]
Your devices don't won't to pair, because they think they are on the network already

```
debug 2023-10-23 12:48:36: Received MQTT message on 'zigbee2mqtt/0x84b4dbfffeff41c1/availability' with data '{"state":"online"}'
debug 2023-10-23 12:49:01: Received MQTT message on 'zigbee2mqtt/0x84b4dbfffeff41c1/set' with data 'ON'
error 2023-10-23 12:49:01: Entity '0x84b4dbfffeff41c1' is unknown
```
Device sends availability -> is on the network
Error unknown -> is not in the database

I don't know how to solve this, but I would start by changing the panid and network-key
